model/pom.xml

```diff
@@ -41,6 +41,11 @@
             <version>1.10.19</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.apache.commons</groupId>
+            <artifactId>commons-lang3</artifactId>
```

Comment:
We are not using maven anymore (there is a jira task to cleanup that). Can you edit gradle file please.